src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react';
+import {useDraftObservation} from '../../hooks/useDraftObservation';
+import {defineMessages, useIntl} from 'react-intl';
+import {EditScreen} from '../../sharedComponents/EditScreen';
+import {PresetCircleIcon} from '../../sharedComponents/icons/PresetIcon';
+import {usePersistedDraftObservation} from '../../hooks/persistedState/usePersistedDraftObservation';
+import {NativeNavigationComponent} from '../../sharedTypes/navigation';
+import {useCreateObservation} from '../../hooks/server/observations';
+import {CommonActions} from '@react-navigation/native';
+import {Photo, DraftPhoto} from '../../contexts/PhotoPromiseContext/types';
+import {useCreateBlobMutation} from '../../hooks/server/media';
+import {usePersistedTrack} from '../../hooks/persistedState/usePersistedTrack';
+import {SaveButton} from '../../sharedComponents/SaveButton';
+import {useMostAccurateLocationForObservation} from '../ObservationEdit/useMostAccurateLocationForObservation';
+
+const m = defineMessages({
+  observation: {
+    // Keep id stable for translations
```

Comment:
fun fact, the translations are shared between repos. We should have done a better job at this, but if they have the same id and the same strings as mapeo, it will just take the mapeo translations. Hence this comment

Reply:
ah good to know, although that doesn't seem ideal. maybe i'm missing some context for why it's useful to have it set up that way

Reply:
Its just avoids us having to do the translations twice. Most of the strings in CoMapeo and Mapeo are identical, so there are many translations that we can just sync up.

Reply:
seems kind of fragile to me, especially since this isn't documented yet and hasn't been mentioned until now. wondering if crowdin provides tooling for copying over translations or something like that so that the burden of re-doing translations is on us and not actual translators, if we wanted to create a separate project for this

Reply:
Im actually making it seem more fragile than it is. So mapeo (legacy) and CoMapeo share a [translation-memory](https://support.crowdin.com/translation-memory/). Which means that any translated strings (separate of the ID) that are the same, will automatically be detected and translated.

When i talked to crowdin the support, they also told me that if id's are the same, it would make it easier for the system to match them. They are not actually tied to the same translation, the id are just used to help match any translations. But if we overwrite the ids or strings, it won't effect the other app unless they are identical 'defaultMessages`. And in that case it is up to the final reviewer (an AD team member ) to determine whether the translation is overwritten or both translations should exist in either app.

Reply:
this is very helpful context - thanks for explaining! can this be documented somewhere? (whether it's in the repo docs or an issue)